package.json

```diff
@@ -1,37 +1,45 @@
 {
-  "name": "jellyfin-theater",
-  "productName": "Jellyfin Theater",
+  "name": "jellyfin-desktop",
   "version": "3.0.5",
```

Comment:
We can probably change this to something else (Bring it back to 1.0?)

Reply:
Yeah